### PR TITLE
Reformat exception to better understand the source of the problem

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Reformat exception to better understand the source of the problem.

### What issues does this PR fix or reference?
Hard to understand the reason of errors like this:
```
Failed to parse last timestamp of the event. Cause: For input string: "". Event: PodEvent{podName='XXX', containerName='go-cli', reason='Created', message='Created container', creationTimestamp='2020-01-01T00:28:56Z', lastTimestamp='2020-01-01T00:28:56Z'}
```

#### Release Notes
n/a

#### Docs PR
n/a

### CODE REVIEW

**Review by ibuziuk — APPROVED** [review]

File: infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesDeployments.java [hunk 1]
```diff
@@ -530,11 +530,12 @@ public void eventReceived(Action action, Event event) {
                   if (happenedAfterWatcherInitialization(podEvent)) {
                     containerEventsHandlers.forEach(h -> h.handle(podEvent));
                   }
-                } catch (ParseException | IllegalArgumentException e) {
+                } catch (ParseException e) {
                   LOG.error(
-                      "Failed to parse last timestamp of the event. Cause: {}. Event: {}",
-                      e.getMessage(),
-                      podEvent);
+                      String.format(
```
Comment (sparkoo) [inline infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesDeployments.java]: why change to `String.format` ?

Comment (skabashnyuk) [reply infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesDeployments.java]:
```
2020-01-03 09:20:46,486[main]             [ERROR] [.w.i.k.n.KubernetesDeployments 12]   - Failed to parse last timestamp of the event. Cause: Message. Event: PodEvent{podName='podName', containerName='containerName', reason='reason', message='message', creationTimestamp='time', lastTimestamp='time2'}
```
vs
```
2020-01-03 09:20:46,498[main]             [ERROR] [.w.i.k.n.KubernetesDeployments 17]   - Failed to parse last timestamp of the event. Cause: Message. Event: PodEvent{podName='podName', containerName='containerName', reason='reason', message='message', creationTimestamp='time', lastTimestamp='time2'}
java.lang.Exception: Message
	at Scratch.main(scratch_4.java:10)
```
in this case, when we use (string, Exception) constructor we have also stacktrace printed.

Comment (sparkoo) [reply infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesDeployments.java]: Ah, I didn't notice extra `e` arg. Anyway, it should be enough to just add `e` into `LOG.error` http://slf4j.org/faq.html#paramException

Comment (skabashnyuk) [reply infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesDeployments.java]: @sparkoo you are right.

**Review by sparkoo — APPROVED** [review]

**Pull request merged**